interfaces/builtin/screen_inhibit_control.go

```diff
@@ -26,6 +26,7 @@ import (
 const screenInhibitControlConnectedPlugAppArmor = `
 # Description: Can inhibit and uninhibit screen savers in desktop sessions.
 #include <abstractions/dbus-session-strict>
+#include <abstractions/dbus-strict>
```

Comment:
This in and of itself is quite safe because it doesn't allow connecting to any services on the system bus, but then that begs the question-- if no services are accessed on the system bus, why do you need to use this abstraction? Can you provide more details on why this is needed along with other details regarding your environment (eg, do you have other interfaces connected, etc). Thanks!